src/module/system/damage/dialog.ts

```diff
@@ -45,7 +48,10 @@ class DamageModifierDialog extends Application {
         this.baseDamageType = params.formulaData.base.at(0)?.damageType ?? "untyped";
         this.degree = DEGREE_OF_SUCCESS_STRINGS.indexOf(this.context.outcome ?? "success") as DegreeOfSuccessIndex;
 
-        this.#originallyEnabled = new Set(this.formulaData.modifiers.filter((m) => m.enabled));
+        this.#originallyDisabled = {
```

Comment:
```suggestion
        this.#originallyDisabled = {
```
Since you flipped this from `originallyEnabled` to `originallyDisabled`, all the rest needs to follow.

Reply:
I interpreted your earlier review comment as a request to swap the name alongside adding properties. I've swapped it back to the original and changed all references.